cms/envs/common.py

```diff
@@ -492,6 +492,7 @@
 
 ############################# MICROFRONTENDS ###################################
 COURSE_AUTHORING_MICROFRONTEND_URL = None
+DISCUSSION_MICROFRONTEND_URL = None
```

Comment:
Currently there seems to be no discussion section in studio and thus this url setting is unused. 
Is this url setting required for a future scope ?

Reply:
Yes, there are future scenarios where that will be needed. Including, when the discussions block is loaded in studio.